docker/gitea/versions.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Google LLC
+#
+# This program is free software; you can redistribute it and/or modify
```

Comment:
This header seems to be a GPL 2 one. Gitea is MIT

Reply:
You can check headers here: https://opensource.google/documentation/reference/releasing/licenses